packages/flutter/test/material/tabs_test.dart

```diff
@@ -2350,4 +2350,94 @@ void main() {
     expect(find.text('Tab1'), findsOneWidget);
     expect(find.text('Tab2'), findsOneWidget);
   });
+
+    testWidgets('DefaultTabController with zero tabs',
```

Comment:
It also seems that the indentations here are off by 2 spaces compared to the rest of the file.